### DISCUSSION TITLE
f-droid still not installing the latest version 11.5.0

### DISCUSSION THREAD

**Discussion by Josegithub2023** [plateaukao/einkbro#349] [General]
I need to see if the bug in version 11.4.0 is fixed in the latest version i.e. 11.5.0. I can't use any version of einkbro downloaded on Github on my Lenovo Smart paper device

**Comment by plateaukao** [comment]
Now it has latest versions: https://f-droid.org/repo/info.plateaukao.einkbro_110700.apk
